apps/api/src/routers/user.py

```diff
@@ -25,6 +25,13 @@ class IdentityResponse(BaseModel):
     role: Union[Role, None] = None
 
 
+def getHasDeadlinePassed() -> bool:
```

Comment:
Chore: Python names are usually snake_case. Additionally, including `get` is unnecessary, and if this function isn't used outside of this module, we can denote it as private: consider `_is_past_deadline` (and update the test mocks accordingly).